hwdetect/hwdetect.sh

```diff
@@ -0,0 +1,43 @@
+#!/bin/bash
+
+file_orig=file.txt
+file_cmp=new_file.txt
```

Comment:
Consider placing temporary files under /tmp folder and deleting them after script exit.
What if filename already exists and currently in use?
Need to add check for this case and generate another random name if needed.